src/handle-data.js

```diff
@@ -5,6 +5,21 @@ const directionMap = {
 
 const MAX_CELLS = 10000;
 
+/**
+ * Get total cell info
+ * @param {Boolean} isDim
+ * @param {Object} layout
+ * @param {Number} colIndex
+ * @param {Number} numDims
+ * @returns dimensions and measures total cell values as strings
+ */
+export function getTotalInfo(isDim, layout, colIndex, numDims) {
+  if (!isDim) return layout.qHyperCube.qGrandTotalRow[colIndex - numDims]?.qText;
+  if (colIndex !== 0) return '\u00A0';
+  if (layout.totals.label !== undefined && layout.totals.label) return layout.totals.label;
+  return 'Totals';
```

Comment:
@Max when the condition reaches to line 20, it supposed to return `translator.get('Object.Tabl.Totals')` , but without translator it just returns the input as a text. I just replace it with Totals but not sure if it's a good idea ....

Reply:
It should be `'Object.Tabl.Totals'` here, then it will be translated later in the totals component